db/pgseed.js

```diff
@@ -0,0 +1,42 @@
+require('dotenv').config();
+
+const { Pool, Client } = require('pg');
+// pools will use environment variables
+// for connection information
+const client = new Client({
+  user: 'jingchen',
```

Comment:
i think would be a good idea to put this information into a file, then gitignore.